src/applications/ivc-champva/shared/components/fileUploads/MissingFileList.jsx

```diff
@@ -9,11 +9,15 @@ import { makeHumanReadable } from '../../utilities';
  * @param {object} file Object representing a missing file
  * @param {string} entryName String of a person's name
  * @param {number} index entry index number (used to target list-loop element)
- * @param {object} fileNameDict (optional) Mapping of file names to arbitrary display text
+ * @param {object} fileNameMap (optional) Mapping of file names to arbitrary display text
  * @returns JSX
  */
-function alertOrLink(file, entryName, index, fileNameDict = {}) {
-  const fn = fileNameDict?.[file.name] ?? makeHumanReadable(file.name);
+function alertOrLink(file, entryName, index, fileNameMap = {}) {
+  // The fileNameMapvalue may be an object if the particular file in
+  // question has a nested `name` (see 10-10D constants.js file for example)
+  let fn = fileNameMap?.[file.name] ?? makeHumanReadable(file.name);
+  fn = typeof fn !== 'string' && fn.name ? fn.name : fn;
+
```

Comment:
This is what lets us use `REQUIRED_FILES` items that are either just a string, OR that may contain an extra `href` we'd like to display.